### ISSUE TITLE
508/a11y: Methods & Evaluation Resources page - multiple H1 elements

### ISSUE THREAD

**Issue by iamjolly** [gsa-oes/office-of-evaluation-sciences#342]
Page: [Methods & Evaluation Resources](https://federalist-9dc074d2-9bb0-4720-8e37-036899ae1463.app.cloud.gov/preview/gsa-oes/office-of-evaluation-sciences/2021-website-updates-2.0/methods/)

This page has four (4) `h1` headings on the page.

Each page should have only one main heading (`h1`). 

From the [18F Accessibility Guide on Headings](https://accessibility.18f.gov/headings/):
> When using heading elements, reserve the `<h1>` element for the page title. On the home page, this is usually the title of the site and on other pages, this may be the page title. Use the `<h1>` element once per page—some assistive technology may be unable to read multiple `<h1>`s on a single page correctly. Other heading levels may be used more than once following document outline order.

Recommendation: 
Change the `h1` elements and corresponding heading structure so the headings have consistent hierarchy so that the page title (currently the first `h1`) is the only `h1` on the page.

**Comment by alyssagalik** [comment]
updated headings to h2 and h3
